package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bootbot",
-  "version": "1.0.9",
+  "version": "1.1.0",
```

Comment:
Can you revert this, please? We'll handle the new version during the publish/release process.